Makefile

```diff
@@ -11,22 +11,63 @@ run-with-local:
 test:
 	pipenv run python setup.py test --addopts -vs
 
+# namespace/project where all the deployments will be made
+NAMESPACE=lad
+# route for the Factstore deployed
+FACTSTORE_ROUTE="http://LAD.FACTSTORE.URL.ENTER.HERE.com/"
```

Comment:
Seeing how the factstore is included as part of an openshift deployment stack, I think it'd be a good idea to show the format of the service at least in a comment, i.e.: 
http://<service_addr>:<service_port>

Reply:
Nvm, I see now that this is meant for a new Route created via deployment

Reply:
:+1: